src/test/fixtures/profiles/processed-profile.js

```diff
@@ -119,18 +175,107 @@ export function getThreadWithMarkers(markers: TestDefinedMarkers) {
   return thread;
 }
 
+export function getThreadWithRawMarkers(markers: TestDefinedRawMarker[]) {
+  const thread = getEmptyThread();
+  addRawMarkersToThread(thread, markers);
+  return thread;
+}
+
+/**
+ * This can be a little annoying to derive with all of the dependencies,
+ * so provide an easy interface to do so here.
+ */
+export function getTestFriendlyDerivedMarkerInfo(thread: Thread) {
+  return deriveMarkersFromRawMarkerTable(
+    thread.markers,
+    thread.stringTable,
+    thread.tid || 0,
+    getTimeRangeForThread(thread, 1),
+    new IPCMarkerCorrelations()
+  );
+}
+
+// The following functions are helpers to make TestDefinedRawMarker. They are someone
+// intentionally terse since they are used frequently in long lists of markers.
+
+export function makeStart(
```

Comment:
optional: I see why they are pretty terse, but I also feel like it would be better to make them a bit more explicit, at least about *what* it makes, like `makeStartMarker`. but then it doesn't be short anymore 🤔. So I'll leave that to you.